src/ast/node/pkg.rs

```diff
@@ -21,19 +21,25 @@ use super::PrintTrait;
 use super::{primary::VarNode, Node, NodeResult};
 #[node]
 pub struct UseNode {
-    pub ids: Vec<Box<VarNode>>,
-    /// 是否完整
-    /// use a::b 完整
-    /// use a::b:: 不完整
-    pub complete: bool,
-    pub singlecolon: bool,
+    /// namespace imported by use keyword
+    pub namespace: Vec<Box<VarNode>>,
+
+    /// whether the pub modifier exists
     pub modifier: Option<(TokenType, Range)>,
+
+    /// whether the namespace is imported fully
     pub all_import: bool,
```

Comment:
the expression here seems strange, I would say "whether import all symbols in the namespace"